email/email.go

```diff
@@ -33,7 +34,9 @@ func NewSender(smtpHost string, smtpUsername string, smtpPassword string,
 	}
 	smtpURL += smtpHost
 
-	tlsConfig := tls.Config{}
+	host, _, _ := net.SplitHostPort(smtpHost)
```

Comment:
I think it would be better not to ignore errors. Maybe log it and if no error set the ServerName.

Reply:
well, in worse case `net.SplitHostPort` will return  `smtpHost` as is
https://play.golang.org/p/etsLIZv_Oz8

but you're right providing smtpHost without port it wouldn't work anyway
so, error checking will prevent it